commit a520181562c2ab6514aa12f759bad9712f20d45c (refs/changes/17/1417/1)
Author: andrew <andrew@whartonlabs.com>
Date:   2017-02-03T11:47:52+11:00 (2 years, 8 months ago)
    
    Change argument check from String to UUID
    
    The package should always be specified as its id (UUID) when calling the CloudApi. The normalisation of name/shortId/id to id is handled in TritonApi.

diff --git a/lib/cloudapi2.js b/lib/cloudapi2.js
index 90ebecf..f887714 100644
--- a/lib/cloudapi2.js
+++ b/lib/cloudapi2.js
@@ -782,7 +782,7 @@ CloudApi.prototype.getMachine = function getMachine(opts, cb) {
  */
 CloudApi.prototype.resizeMachine = function resizeMachine(opts, callback) {
     assert.uuid(opts.id, 'opts.id');
-    assert.string(opts.package, 'opts.package');
+    assert.uuid(opts.package, 'opts.package');
     var data = {
         action: 'resize',
         package: opts.package
